index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8" />
+<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+<link href="./styles/index.css" rel="stylesheet" />
+<script src="./scripts/index.js"></script>
+<title>Weather Report</title>
+</head>
+<body>
+<!--Wave 2-->
+<div class="header">
+    <h1>Weather Report</h1>
+    <h2>For the loveley city of</h2>
+    <p class="city-name">🌊 🌙 Oceanville🌙🌊</p>
+</div>
```

Comment:
don't forget to use semantic html, like `header`, `section`, `main` rather than `div`